static/app/components/devtoolbar/components/deadClicks/deadClicksPanel.tsx

```diff
@@ -0,0 +1,160 @@
+import {useCallback, useRef} from 'react';
+
+import AnalyticsProvider from 'sentry/components/devtoolbar/components/analyticsProvider';
+import SentryAppLink from 'sentry/components/devtoolbar/components/sentryAppLink';
+import useConfiguration from 'sentry/components/devtoolbar/hooks/useConfiguration';
+import {
+  badgeWithLabelCss,
+  listItemGridCss,
+  listItemPlaceholderWrapperCss,
+} from 'sentry/components/devtoolbar/styles/listItem';
+import {smallCss} from 'sentry/components/devtoolbar/styles/typography';
+import Placeholder from 'sentry/components/placeholder';
+import TextOverflow from 'sentry/components/textOverflow';
+import type {DeadRageSelectorItem} from 'sentry/views/replays/types';
+
+import {panelInsetContentCss, panelSectionCss} from '../../styles/panel';
+import {resetFlexColumnCss} from '../../styles/reset';
+import InfiniteListItems from '../infiniteListItems';
+import InfiniteListState from '../infiniteListState';
+import PanelLayout from '../panelLayout';
+
+import useInfiniteDeadClicksList from './useInfiniteDeadClicksList';
+
+export default function DeadClicksPanel() {
+  const queryResult = useInfiniteDeadClicksList({
+    sort: '-count_dead_clicks',
+  });
+
+  const estimateSize = 89;
+  const placeholderHeight = `${estimateSize - 8}px`; // The real height of the items, minus the padding-block value
+
+  return (
+    <PanelLayout title="Dead Clicks">
+      <div css={resetFlexColumnCss}>
+        <InfiniteListState
+          queryResult={queryResult}
+          backgroundUpdatingMessage={() => null}
+          loadingMessage={() => (
+            <div
+              css={[
+                resetFlexColumnCss,
+                panelSectionCss,
+                panelInsetContentCss,
+                listItemPlaceholderWrapperCss,
+              ]}
+            >
+              <Placeholder height={placeholderHeight} />
+              <Placeholder height={placeholderHeight} />
+              <Placeholder height={placeholderHeight} />
+              <Placeholder height={placeholderHeight} />
+            </div>
+          )}
+        >
+          <InfiniteListItems
+            estimateSize={() => estimateSize}
+            queryResult={queryResult}
+            itemRenderer={props => <DeadClickListItem {...props} />}
+            emptyMessage={() => <p css={panelInsetContentCss}>No items to show</p>}
+          />
+        </InfiniteListState>
+      </div>
+    </PanelLayout>
+  );
+}
+
+function DeadClickListItem({item}: {item: DeadRageSelectorItem}) {
+  const {projectId} = useConfiguration();
+
+  const {onMouseOver, onMouseOut} = useHighlightMouseEvents({
+    item,
+  });
+  return (
+    <AnalyticsProvider keyVal="issue-list.item" nameVal="issue list item">
```

Comment:
```suggestion
    <AnalyticsProvider keyVal="dead-click-list.item" nameVal="dead click list item">
```
not sure if we want to rename this since they're not exactly issue items